firstapp/app/src/androidTest/java/com/example/firstapp/ShellUtility.java

```diff
@@ -22,18 +23,23 @@
 
 
 public class ShellUtility {
+    public static UiDevice device;
```

Comment:
Should avoid storing static variables in your methods.  One solution is to make its methods be non-static.  This would mean callers would need to do the following:

    ShellUtility shellUtility = new ShellUtility();
    shellUtility.doSomething()

Another solution is have callers store the UiDevice, and not have that responsibility fall on ShellUtility.